samples/artifacts/Data Transformation/MapExtensionSample.siddhi

```diff
@@ -0,0 +1,54 @@
+@APP:name("MapExtensionSample")
+
+@App:description('Create a map, insert values into the map and checks if the given object is a map or not')
+
+/*
+
+Purpose:
+	This function creates a map, add values and checks whether the given object is a map or not
+
+Prerequisites:
+    * Save this sample. If there is no syntax error, the following messages would be shown on the console
+	     * - Siddhi App MapExtensionSample successfully deployed.
+
+Executing the Sample:
+	1) Start the Siddhi application by clicking on 'Run'
+	2) If the Siddhi application starts successfully, the following messages would be shown on the console
+            * MapExtensionSample.siddhi - Started Successfully!
+
+Testing the Sample:
+  You can publish data event to the file, through event simulator
+        a) Open event simulator by clicking on the second icon or press Ctrl+Shift+I.
+    	b) In the Single Simulation tab of the panel, select values as follows:
+                * Siddhi App Name  : MapExtensionSample
+                * Stream Name     : SweetProductionStream
+        c) Enter following values in the fields and send
+                name: chocolate cake
+                amount: 50.50
+
+Viewing the Results:
+    Messages similar to the following would be shown on the console.
+        INFO {org.wso2.siddhi.core.query.processor.stream.LogStreamProcessor} - MapExtensionSample: Event: , StreamEvent{ timestamp=1513384974698, beforeWindowData=null, onAfterWindowData=null, outputData=[true, false], type=CURRENT, next=null}
+
+
+*/
+
+@sink(type='log')
+define stream CheckedMapStream(isMap1 bool, isMap2 bool);
+
+define stream SweetProductionStream (name string, amount double);
+
+@info(name = 'query1')
+from SweetProductionStream
+select name, amount, map:create() as tmpMap
+insert into tmpStream;
+
+@info(name = 'query2')
```

Comment:
Change the query name to something more context specific.In this case something like "map-creator"